app/scripts/modules/appengine/serverGroup/configure/wizard/basicSettings.html

```diff
@@ -113,15 +138,7 @@ <h5 class="text-center">
             <div class="text-left"
                  ng-if="!basicSettingsCtrl.createsNewCluster() && command.viewState.mode === 'create' && latestServerGroup">
               <p>
-                There is already a server group in this cluster. Do you want to clone it?
-              </p>
-              <p>
-                Cloning copies the entire configuration from the selected server group, allowing you
-                to modify whichever fields (e.g. image) you need to change in the new server group.
-              </p>
-              <p>
-                To clone a server group, select "Clone" from the "Server Group Actions" menu in the details view of the
-                server group.
+                There is already a server group in this cluster.
```

Comment:
I don't think there will be a clone operation for App Engine, which is why I removed these lines. But I think that would be best done in a different PR.